lib/beetle/subscriber.rb

```diff
@@ -168,45 +168,44 @@ def resume(queue_name)
     def create_subscription_callback(queue_name, amqp_queue_name, handler, opts)
       server = @server
       lambda do |header, data|
+        msg_id = header.attributes[:message_id]
         if channel(server).closing?
-          logger.info "Beetle: ignoring message #{header.attributes[:message_id]} (#{header.attributes[:timestamp]}) since channel to server #{server} already closed"
+          timestamp = header.attributes[:timestamp]
+          logger.info "Beetle: ignored message #{msg_id}(#{timestamp}) since channel to server #{server} was already closed"
           return
         end
         begin
-          logger.debug("Beetle: #{header.attributes[:message_id]} (#{header.attributes[:timestamp]}) received")
```

Comment:
Shouldn't we keep the *received* log but place it after the `pre_process` call?

Reply:
The log line is in the message.process method.